src/main/resources/db/changelog/changesets/changelog_20231103T151738Z.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-latest.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet author="marcelinhug (generated)" id="1699024666101-1">
+        <addColumn tableName="voltage_limit">
+            <column name="limit_type" type="varchar(255)" />
+        </addColumn>
+    </changeSet>
+    <changeSet author="marcelinhug" id="1699024666101-2">
+        <update tableName="voltage_limit">
+            <column name="limit_type" value="MODIFICATION"/>
```

Comment:
Why not a defaultValue for the new column instead of this update ?

Reply:
It didn't work out for me when I tried it, nevertheless I think setting a default value doesn't make much sense for the column in the long run since it's only needed for the data migration